homework01/rudolf/vector.cpp

```diff
@@ -0,0 +1,46 @@
+#include "vector.h"
+
+Vector::Vector()
+{
+	mX=0;
+	mY=0;
+}
+
+Vector::Vector(double x, double y): mX(x), mY(y){}
+
+double Vector::dotProduct(const Vector& vec) const
```

Comment:
Good to see the argument passed as const ref and the constness of this method! It should be `DotProduct` though ;)